### PR TITLE
Turkish language integration added to required files

### PR DESCRIPTION
**Pull request by Clas0512**
tr.json file added to /angular/src/assets/i18n/ folder.
Added a line about Turkish shortcut to /angular/src/app/header/header.component.html:24.
Added a node for language list in /angular/src/app/app.component.ts:37

### CODE REVIEW

**Review by axelcoezard — APPROVED** [review]
The code is ok for me :+1:

**Review by axelcoezard — COMMENTED** [review]
@d-r-e Can you check this PR and validate it ?

File: angular/src/app/header/header.component.html [hunk 1]
```diff
@@ -21,6 +21,7 @@
             <option value="en" [selected]="currentLanguage === 'en'">🇬🇧 English</option>
             <option value="fr" [selected]="currentLanguage === 'fr'">🇫🇷 Français</option>
             <option value="es" [selected]="currentLanguage === 'es'">🇪🇸 Español</option>
+            <option value="tr" [selected]="currentLanguage === 'tr'">🇪🇸 Türkçe</option>
```
Comment (axelcoezard) [inline angular/src/app/header/header.component.html:24]: wrong flag

**Review by axelcoezard — CHANGES_REQUESTED** [review]

File: angular/src/app/header/header.component.html (hunk 1 above)
Comment (Clas0512) [reply angular/src/app/header/header.component.html]: okay i am changing now

**Review by axelcoezard — APPROVED** [review]
Ok, everything is ok, even the flag !

**Comment by d-r-e** [comment]
Deployed and up!

**Pull request merged**